python/tests/test_aac/plugins/check/test_check_aac.py

```diff
@@ -38,8 +38,21 @@ def test_cli_check(self):
             check_args = [temp_aac_file_path]
 
             exit_code, output_message = self.run_check_cli_command_with_args(check_args)
+            self.assertEqual(0, exit_code, f"Expected success but failed with message: {output_message}")  # asserts the command ran successfully
+            self.assertNotIn("My plugin was successful.", output_message) # only appears when --verbose is passed in.
 
+    def test_cli_check_verbose(self):
```

Comment:
Missed adding this Friday, apologies. But need a test for verbose failure.

Reply:
This cannot be accomplished currently, found gap that `--verbose` does not have handling for failure cases.